Linking errors

I'm trying to build and install aitrack on an arch linux system, and I'm getting a numbers of errors during the build() step. This is the initial output of `makepkg -sri`:
<details>
<summary>makepkg output</summary>

```
==> Making package: aitrack 0.6.4_alpha-1 (Tue 17 Aug 2021 08:32:11 PM CEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating aitrack-linux git repo...
Fetching origin
==> Validating source files with sha256sums...
    aitrack-linux ... Skipped
==> Extracting sources...
  -> Creating working copy of aitrack-linux git repo...
Reset branch 'makepkg'
==> Starting prepare()...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   639  100   639    0     0   2724      0 --:--:-- --:--:-- --:--:--  2719
100 3699k  100 3699k    0     0  4063k      0 --:--:-- --:--:-- --:--:-- 4063k
gzip: onnxruntime-linux-x64-1.4.0.tar already exists; do you wish to overwrite (y or n)? y
==> Removing existing $pkgdir/ directory...
==> Starting build()...
Makefile:629: warning: overriding recipe for target 'qrc_Resource.cpp'
Makefile:624: warning: ignoring old recipe for target 'qrc_Resource.cpp'
g++ -Wl,-O1 -o aitrack data.o filters.o imageprocessor.o model.o PositionSolver.o Main.o CameraFactory.o CameraSettings.o OCVCamera.o Config.o UDPSender.o UpdateChecker.o presenter.o TrackerFactory.o TrackerWrapper.o ConfigWindow.o WindowMain.o qglobalshortcut.o qglobalshortcut_x11.o qrc_Resource.o moc_UpdateChecker.o moc_ConfigWindow.o moc_WindowMain.o moc_qglobalshortcut.o   -L onnxruntime-linux-x64-1.4.0/lib -lonnxruntime -fopenmp -lopencv_gapi -lopencv_stitching -lopencv_alphamat -lopencv_aruco -lopencv_barcode -lopencv_bgsegm -lopencv_bioinspired -lopencv_ccalib -lopencv_cvv -lopencv_dnn_objdetect -lopencv_dnn_superres -lopencv_dpm -lopencv_face -lopencv_freetype -lopencv_fuzzy -lopencv_hdf -lopencv_hfs -lopencv_img_hash -lopencv_intensity_transform -lopencv_line_descriptor -lopencv_mcc -lopencv_quality -lopencv_rapid -lopencv_reg -lopencv_rgbd -lopencv_saliency -lopencv_stereo -lopencv_structured_light -lopencv_phase_unwrapping -lopencv_superres -lopencv_optflow -lopencv_surface_matching -lopencv_tracking -lopencv_highgui -lopencv_datasets -lopencv_text -lopencv_plot -lopencv_videostab -lopencv_videoio -lopencv_viz -lopencv_wechat_qrcode -lopencv_xfeatures2d -lopencv_shape -lopencv_ml -lopencv_ximgproc -lopencv_video -lopencv_dnn -lopencv_xobjdetect -lopencv_objdetect -lopencv_calib3d -lopencv_imgcodecs -lopencv_features2d -lopencv_flann -lopencv_xphoto -lopencv_photo -lopencv_imgproc -lopencv_core -lspdlog -pthread -lfmt -lX11-xcb -lX11 -lxcb -lQt5Widgets -lQt5Network -lQt5X11Extras -lQt5Gui -lQt5Core /usr/lib/libQt5Widgets.so /usr/lib/libQt5Gui.so /usr/lib/libQt5Core.so -lGL -lpthread
/usr/bin/ld: warning: libhdf5.so.200, needed by /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so, not found (try using -rpath or -rpath-link)
/usr/bin/ld: warning: libGLEW.so.2.2, needed by /usr/lib/libvtkRenderingOpenGL2.so.1, not found (try using -rpath or -rpath-link)
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dread'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetNamedRenderbufferParameteriv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewEndTransformFeedback'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fcreate'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__GLEW_VERSION_3_2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_UCHAR_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_size'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDeleteRenderbuffers'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewTexImage3D'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__GLEW_ARB_transform_feedback3'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5check_version'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBeginQuery'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_array_dims2'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGenFramebuffers'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform1f'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetFramebufferAttachmentParameteriv'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tclose'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGenVertexArrays'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewMapBufferRange'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDeleteVertexArrays'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewTexBuffer'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dopen2'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewCheckFramebufferStatus'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetBufferPointerv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewShaderSource'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDeleteFramebuffers'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5open'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sset_extent_simple'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewFramebufferRenderbuffer'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__GLEW_ARB_instanced_arrays'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetAttribLocation'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewVertexAttribPointer'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetQueryObjectiv'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_INT32_g'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewEndQuery'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUseProgram'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_SCHAR_g'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__GLEW_VERSION_3_1'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Screate_simple'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aget_type'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetProgramiv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetQueryObjectui64v'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `glewGetErrorString'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tcreate'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewCreateProgram'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBindRenderbuffer'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Awrite'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tset_strpad'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Gclose'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewCompileShader'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aopen'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_class'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Gcreate2'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewMapBuffer'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBlendEquationSeparate'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewRenderbufferStorageMultisample'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBeginQueryIndexed'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetProgramInfoLog'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewLinkProgram'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUnmapBuffer'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDrawRangeElements'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `glewIsSupported'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aread'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dclose'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewEnableVertexAttribArray'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pset_deflate'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetTextureLevelParameteriv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDetachShader'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Adelete'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewActiveTexture'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5P_CLS_DATASET_CREATE_ID_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Acreate2'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGenRenderbuffers'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dget_create_plist'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBindBufferBase'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBindBuffer'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aopen_name'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Eset_auto2'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBufferData'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform3fv'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sselect_hyperslab'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pset_chunk'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniformMatrix3fv'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pget_layout'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5P_CLS_LINK_ACCESS_ID_g'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDisableVertexAttribArray'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewTransformFeedbackVaryings'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pget_chunk'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBindFragDataLocation'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__GLEW_ARB_texture_float'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetShaderiv'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tinsert'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tequal'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBlitFramebuffer'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewFramebufferTexture2D'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dget_space'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewClampColor'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDeleteProgram'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aget_space'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetQueryObjectuiv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__GLEW_ARB_direct_state_access'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fis_hdf5'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__GLEW_ARB_gpu_shader5'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBeginTransformFeedback'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dget_type'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_FLOAT_g'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDeleteShader'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewQueryCounter'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_USHORT_g'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDrawBuffers'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform1fv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDrawArraysInstancedARB'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pcreate'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Eget_auto2'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBindVertexArray'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tset_size'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBindFramebuffer'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewEndQueryIndexed'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewFramebufferTexture3D'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetUniformLocation'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tarray_create2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sget_simple_extent_ndims'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewRenderbufferStorage'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetShaderInfoLog'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform2iv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform2fv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGenBuffers'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Screate'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniformMatrix4fv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGenerateMipmap'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dwrite'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewTexImage2DMultisample'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dextend'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_super'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_native_type'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDeleteBuffers'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Lexists'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_INT_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_C_S1_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tcopy'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform1i'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewVertexAttribDivisorARB'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewAttachShader'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetRenderbufferParameteriv'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_SHORT_g'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform4fv'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_DOUBLE_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sget_simple_extent_dims'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dcreate2'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewUniform1iv'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewCreateShader'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGetStringi'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewGenQueries'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fopen'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDrawElementsInstancedARB'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewBlendFuncSeparate'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `glewInit'
/usr/bin/ld: /usr/lib/libvtkRenderingOpenGL2.so.1: undefined reference to `__glewDeleteQueries'
collect2: error: ld returned 1 exit status
make: *** [Makefile:291: aitrack] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
```
</details>

I've been able to get rid of the glew related messages by installing the `glew` package, so maybe that's a missing runtime or build dependency?

There were however still some errors left and I don't know how to approach them. Any help would be appreciated. This is the final command output:

<details>
<summary>makepkg output</summary>

```
==> Making package: aitrack 0.6.4_alpha-1 (Tue 17 Aug 2021 08:33:59 PM CEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating aitrack-linux git repo...
Fetching origin
==> Validating source files with sha256sums...
    aitrack-linux ... Skipped
==> Extracting sources...
  -> Creating working copy of aitrack-linux git repo...
Reset branch 'makepkg'
==> Starting prepare()...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   639  100   639    0     0  10401      0 --:--:-- --:--:-- --:--:-- 10306
100 3699k  100 3699k    0     0  6739k      0 --:--:-- --:--:-- --:--:-- 6739k
gzip: onnxruntime-linux-x64-1.4.0.tar already exists; do you wish to overwrite (y or n)? y
==> Removing existing $pkgdir/ directory...
==> Starting build()...
Makefile:629: warning: overriding recipe for target 'qrc_Resource.cpp'
Makefile:624: warning: ignoring old recipe for target 'qrc_Resource.cpp'
g++ -Wl,-O1 -o aitrack data.o filters.o imageprocessor.o model.o PositionSolver.o Main.o CameraFactory.o CameraSettings.o OCVCamera.o Config.o UDPSender.o UpdateChecker.o presenter.o TrackerFactory.o TrackerWrapper.o ConfigWindow.o WindowMain.o qglobalshortcut.o qglobalshortcut_x11.o qrc_Resource.o moc_UpdateChecker.o moc_ConfigWindow.o moc_WindowMain.o moc_qglobalshortcut.o   -L onnxruntime-linux-x64-1.4.0/lib -lonnxruntime -fopenmp -lopencv_gapi -lopencv_stitching -lopencv_alphamat -lopencv_aruco -lopencv_barcode -lopencv_bgsegm -lopencv_bioinspired -lopencv_ccalib -lopencv_cvv -lopencv_dnn_objdetect -lopencv_dnn_superres -lopencv_dpm -lopencv_face -lopencv_freetype -lopencv_fuzzy -lopencv_hdf -lopencv_hfs -lopencv_img_hash -lopencv_intensity_transform -lopencv_line_descriptor -lopencv_mcc -lopencv_quality -lopencv_rapid -lopencv_reg -lopencv_rgbd -lopencv_saliency -lopencv_stereo -lopencv_structured_light -lopencv_phase_unwrapping -lopencv_superres -lopencv_optflow -lopencv_surface_matching -lopencv_tracking -lopencv_highgui -lopencv_datasets -lopencv_text -lopencv_plot -lopencv_videostab -lopencv_videoio -lopencv_viz -lopencv_wechat_qrcode -lopencv_xfeatures2d -lopencv_shape -lopencv_ml -lopencv_ximgproc -lopencv_video -lopencv_dnn -lopencv_xobjdetect -lopencv_objdetect -lopencv_calib3d -lopencv_imgcodecs -lopencv_features2d -lopencv_flann -lopencv_xphoto -lopencv_photo -lopencv_imgproc -lopencv_core -lspdlog -pthread -lfmt -lX11-xcb -lX11 -lxcb -lQt5Widgets -lQt5Network -lQt5X11Extras -lQt5Gui -lQt5Core /usr/lib/libQt5Widgets.so /usr/lib/libQt5Gui.so /usr/lib/libQt5Core.so -lGL -lpthread
/usr/bin/ld: warning: libhdf5.so.200, needed by /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so, not found (try using -rpath or -rpath-link)
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dread'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fcreate'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_UCHAR_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_size'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5check_version'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_array_dims2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dopen2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5open'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sset_extent_simple'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_INT32_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_SCHAR_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Screate_simple'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aget_type'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tcreate'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Awrite'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tset_strpad'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Gclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aopen'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_class'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Gcreate2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aread'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pset_deflate'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Adelete'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5P_CLS_DATASET_CREATE_ID_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Acreate2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dget_create_plist'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aopen_name'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Eset_auto2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sselect_hyperslab'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pset_chunk'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pget_layout'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5P_CLS_LINK_ACCESS_ID_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pget_chunk'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tinsert'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tequal'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dget_space'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aget_space'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fis_hdf5'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dget_type'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_FLOAT_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_USHORT_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pcreate'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Eget_auto2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Aclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tset_size'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tarray_create2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Pclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sget_simple_extent_ndims'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Screate'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dwrite'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dextend'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_super'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tget_native_type'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Lexists'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_INT_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_C_S1_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Tcopy'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sclose'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_SHORT_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5T_NATIVE_DOUBLE_g'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Sget_simple_extent_dims'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Dcreate2'
/usr/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/../../../../lib/libopencv_hdf.so: undefined reference to `H5Fopen'
collect2: error: ld returned 1 exit status
make: *** [Makefile:291: aitrack] Error 1
==> ERROR: A failure occurred in build().
    Aborting...
```
</details>

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -7,7 +7,7 @@ url="https://github.com/mdk97/aitrack-linux"
 source=("git+https://github.com/mdk97/aitrack-linux")
 license=("MIT")
 sha256sums=('SKIP')
-depends=("opencv" "fmt" "spdlog" "qt5-base" "qt5-x11extras" "openmp" "vtk")
+depends=("opencv" "fmt" "spdlog" "qt5-base" "qt5-x11extras" "openmp" "vtk" "glew" "hdf5")
 makedepends=("curl" "gzip" "tar" "make" "gcc" "opencv" "fmt" "spdlog" "qt5-base" "qt5-x11extras" "openmp" "vtk")
 install=aitrack.install
 prepare() {

